frontend/src/metabase/api/field.ts

```diff
@@ -1,16 +1,40 @@
 import type {
   FieldId,
-  FieldSearchInput,
+  SearchFieldValuesRequest,
   FieldValue,
-  FieldValuesResult,
+  GetFieldValuesResponse,
+  Field,
+  GetFieldRequest,
+  UpdateFieldRequest,
 } from "metabase-types/api";
 
 import { Api } from "./api";
-import { idTag } from "./tags";
+import { idTag, tag } from "./tags";
 
 export const fieldApi = Api.injectEndpoints({
   endpoints: builder => ({
-    getFieldValues: builder.query<FieldValuesResult, FieldId>({
+    getField: builder.query<Field, GetFieldRequest>({
+      query: ({ id, ...body }) => ({
+        method: "GET",
+        url: `/api/field/${id}`,
+        body,
+      }),
+      providesTags: (response, error, { id }) => [idTag("field", id)],
+    }),
+    updateField: builder.mutation<Field, UpdateFieldRequest>({
+      query: ({ id, ...body }) => ({
+        method: "PUT",
+        url: `/api/field/${id}`,
+        body,
+      }),
+      invalidatesTags: (response, error, { id }) => [
+        idTag("field", id),
+        idTag("field-values", id),
+        tag("database-metadata"),
+        tag("database-id-fields"),
```

Comment:
Unfortunately there is no `db_id` or anything like that present in fields. But I think that's fine. We would still have to invalidate `database-metadata` and `database-id-fields` either for 1 db or all of them. I don't think people would notice any difference in practice because 1) you can only update fields in Admin atm, 2) multi-db cases are rare